docs/community/devstack.md

```diff
@@ -0,0 +1,139 @@
+
+Afterwards, the stack can be created and spun up via:
+
+1. make docker/all
+2. ./compose up
+
+
+You can how reach the api at http://localhost:5001. The makefile targets created a couple test users, namely admin:admin for the primary superuser. This is purely an API stack with no UI. That makes it difficult to test and use github users, so see the next section about adding the UI.
+
+
+### React.js UI
+
+To connect to UI, the default address is http://localhost:8002. If social auth is enabled, you'll be forced to auth via github if clicking on the "login" link at the top right of the page. The presence of SOCIAL_AUTH_GITHUB_KEY & SOCIAL_AUTH_GITHUB_SECRET in the backend configuration triggers dynaconf to set a feature flag for external auth that the UI reads and alters the login link accordingly. If you want to bypass github and use a local django user, go directly to the http://localhost:8002/ui/login url and login that way.
+
+#### docker-compose
+Inside .compose.env, you'll find a commented out line referencing ANSIBLE_HUB_UI_PATH. If you want to add a UI to the compose stack, this needs to be uncommented and set to a valid absolute path for a checkout of [ansible-hub-ui](https://github.com/ansible/ansible-hub-ui). The compose spin up should properly allocate an alpine container with the appropriate node.js version and install+build+launch the UI.
+
+#### starting directly
+
+For development work on the UI, it's easier to launch the webpack dev server outside the compose stack. The npm start scripts and configuration will have it redirect api calls to http://localhost:5001 by default, so no hacking is necessary to make it work locally.
+
+1. Install [NVM](https://github.com/nvm-sh/nvm)
+2. Use NVM to install and set the version to 16. "nvm install 16"
+3. Clone the [ansible-hub-ui](https://github.com/ansible/ansible-hub-ui) repo to the desired location.
+4. From the checkout, run "npm install"
+5. From the checkout, run "npm run start-community"
+
+## Testing
+
+The GalaxyNG project is a "test driven development" (TDD) project. Our interpretation of TDD is that -every- PR to the master branch -must- have some sort of test or test changes. It can be unit, functional or integration, but something must be written. It doesn't matter if the PR is a bugfix or a feature, tests must be written. This is how we avoid regressions and keep track of how new code is meant to work inside a very complex and spread out architecture with many different configurable behaviors.
+
+### Unit
+
+More focus has been spent on the integration tests, but there are a few unit tests in the project. The compose stack must be running to launch unit tests, as they are using django and postgres with a temporary database to do real SQL calls that are incompatible with sqlite.
+
+To launch units, execute "make docker/test/unit". Specific tests can be selected by running "TEST=xyz make docker/test/unit".
```

Comment:
code quotes `make docker/test/unit`

Reply:
Add an example for running specific tests.


For example, to run unit tests for tasks: `TEST=.app.test_tasks make docker/test/unit`